sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -537,8 +536,9 @@ def extract_output(self, accumulator, *args, **kwargs):
     else:
       assert not holds_comparables
 
-    assert len(heap) <= self._n
-    heap.sort(reverse=True)
+    if heap:
```

Comment:
Added null checking to handle empty variables that would've triggered the two assertions removed from this file.
This was the only error to emerge from removing the assertions. It seems that an empty accumulators being passed at lines 501 and 531 is what those assertions were meant to handle, but other than this, the existing logic handles it fine.